crates/gitql-ast/src/function.rs

```diff
@@ -242,6 +250,26 @@ fn text_char(inputs: Vec<Value>) -> Value {
     return Value::Text("".to_string());
 }
 
+fn text_replace(inputs: Vec<Value>) -> Value {
```

Comment:
Why not using the std replace function from string type

```rs
let text = inputs[0].as_text();
let old_string = &inputs[1].as_text();
let new_string = &inputs[2].as_text();
return Value::Text(text.replace(old_string, new_string));
```

Reply:
The search has to be case insensitive, but `.replace` is std is case sensitive. If I use `.replace` this case won't pass.
`SELECT REPLACE("ABC ABC ABC", "a", "c") as replacedText`

Reply:
Ok yes i will search if we can make replace ignore case, and please resolve the conflicts